guides/common/modules/proc_configuring-a-finish-template-for-amazon-ec2.adoc

```diff
@@ -39,25 +39,19 @@ EOS
 . Click the *Templates* tab, and from the *Finish Template* list, select your finish template.
 . In the {ProjectWebUI}, navigate to *Hosts* > *Create Host* and enter the information about the host that you want to create.
 . Click the *Parameters* tab and navigate to *Host parameters*.
-ifdef::satellite,orcharhino[]
+ifdef::katello,satellite,orcharhino[]
 . In *Host parameters*, click the *Add Parameter* button three times to add three new parameter fields.
-Add the following three parameters:
 endif::[]
-ifdef::foreman-el,foreman-deb,katello[]
+ifndef::katello,satellite,orcharhino[]
 . If you have the Remote Execution plugin installed, in *Host parameters*, click the *Add Parameter* button two times to add two new parameter fields.
-If you use the Katello plugin, add a third parameter field.
-Add the following parameters:
 endif::[]
+Add the following parameters:
 .. In the *Name* field, enter `remote_execution_ssh_keys`.
 In the corresponding *Value* field, enter the output of `cat /usr/share/foreman-proxy/.ssh/id_rsa_foreman_proxy.pub`.
```

Comment:
@theforeman/remote-execution is this really needed? I'd think that this is default when REX is installed and the correct REX proxy is assigned.

Reply:
No, it shouldn't be needed

Reply:
https://github.com/theforeman/foreman-documentation/pull/2331 so this PR stays focused and it can get its own discussion.